### PR TITLE
Use IMA videojs plugin for adverts

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
Rather than VAST - this allows better targeting of ads through dfp

Plugin isn't super configurable though, so taken a fork and using that for now

### CODE REVIEW

**Comment by phamann** [comment]
Think I may have caused you a major merge conflict with #7914

File: grunt-configs/requirejs.js [hunk 1]
```diff
@@ -155,14 +155,12 @@ module.exports = function(grunt, options) {
                 name: 'bootstraps/video-player',
                 out: options.staticTargetDir + 'javascripts/bootstraps/video-player.js',
                 paths: {
```
Comment (rich-nguyen) [inline grunt-configs/requirejs.js]: sounds like a good idea to move this into the shared options paths object, due to the lack of support for merging paths references. Will save a lot of developers debugging time.

File: static/src/stylesheets/module/content/_media-player.scss [hunk 2]
```diff
@@ -893,3 +930,64 @@ $vjs-height-ratio: 7/3;
...
+
+    .vjs-mousemoved &,
+    &:hover {
+        bottom: 0;
+        @include transition(bottom 0s);
+        transition-delay: 0s;
+    }
+}
+#ima-seek-bar-div {
+    height: 6px;
```
Comment (rich-nguyen) [inline static/src/stylesheets/module/content/_media-player.scss]: `gs-baseline` ?

File: static/src/javascripts/projects/video/modules/video-init.js [hunk 3]
```diff
@@ -1,7 +0,0 @@
-define([
```
Comment (rich-nguyen) [inline static/src/javascripts/projects/video/modules/video-init.js]: Great, thanks

**Comment by rich-nguyen** [comment]
:+1: super cool

**Pull request merged**